apps/web/src/pages/quick-start/components/SetupStatus.tsx

```diff
@@ -48,15 +44,17 @@ export function SetupStatus({
         </TextStyled>
       </Stack>
 
-      <Group>
-        <Stack spacing={8}>
-          <TextStyled>Don't want to configure the In-App center now?</TextStyled>
-          <TextStyled>Configure it later in a notification workflow builder.</TextStyled>
-        </Stack>
-        <Button variant="outline" onClick={handleConfigureLater}>
-          Configure Later
-        </Button>
-      </Group>
+      <When truthy={onConfigureLater !== undefined}>
```

Comment:
Not sure if being too defensive but do you think would also be needed this is a function?